### ISSUE TITLE
PowerWall Charge and Grid Feed issues

### ISSUE THREAD

**Issue by NicBetts** [datMaffin/homebridge-tesla-powerwall#18]
Updated to the latest and the Battery Charge and Grid Feed show in Homebridge but not in the Apple Home App.

In the Apple Home App they just say Not supported - no value, etc.

Also even if I uncheck favorite they don't hold that choice and continue to remain on Home page.

Help, love this plugin but obviously need those bits to work!

**Comment by datMaffin** [comment]
Your issue sounds really similar to #17 (I assumed the #17 issue was fixed: There was a bug that prevented disabling (unsupported) accessories). The version 1.0.1 includes the fix.

From what version did you update from? (A time frame of when you first installed the plugin would also suffice)

**Comment by NicBetts** [comment]
Apologies for the delay.  Finally my new Raspberry pi 4 arrived today so I decided to build a fresh install of Homebridge and then migrated the config and installed the plugins from new.  It now works.

The previous raspberry pi had the Tesla powerwall plug in from last year (November ish) and I upgraded it this morning.  Thats when I had the issue.   

That said the fresh build seems to have fixed everything.  Love this plugin - thank you
